src/c_api.cpp

```diff
@@ -497,7 +498,8 @@ LAS_DLL LASHeaderH LASReader_GetHeader(const LASReaderH hReader)
 
 
 LAS_DLL LASErrorEnum LASReader_SetOutputSRS(LASReaderH hReader, const LASSRSH hSRS) {
-    
+   
+		using namespace boost::placeholders;
```

Comment:
Nitpick: can we avoid tabs or 8-12 spaces here and indent with 4 spaces?

Reply:
It fixes the same problem I have encountered in the issue.